### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by stephy2001**
nice to meet u . . .

### CODE REVIEW

**Comment by fionamary28** [comment]
yaa...nice to meet u

**Comment by RaichalRose** [comment]
hii girls

**Comment by fionamary28** [comment]
Hook.......what's up man...leave us free

**Comment by RaichalRose** [comment]
i am a girl

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # rae1
 print (hello)
```
Comment (RaichalRose) [inline README.md:2]: something went wrong

**Comment by fionamary28** [comment]
oooh....say frequentlty.....i am forgetting that always

**Comment by RaichalRose** [comment]
fiona ...on the floor

**Pull request merged**